code/modules/surgery/surgery.dm

```diff
@@ -99,22 +99,22 @@
 		return
 	surgery.step_in_progress = 1
 
+	var/speed_mod = 1
```

Comment:
Should probably do a merge from master?

This makes me quite nervous, to say the least. I've seen stuff like this crop up before, and it never ends well.